lib/YaST/Bootloader/BootloaderOptionsPage.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class introduces all accessing methods for Boot loader options tab in bootloader module
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::Bootloader::BootloaderOptionsPage;
+use parent 'Installation::Navigation::NavigationBase';
+use strict;
+use warnings;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init();
+}
+
+sub init {
+    my $self = shift;
+    $self->SUPER::init();
+    $self->{txt_grub_timeout} = $self->{app}->textbox({id => "\"Bootloader::TimeoutWidget\""});
+    return $self;
+}
+
+sub is_shown {
+    my ($self) = @_;
+    $self->{txt_grub_timeout}->value();
+}
+
+sub disable_grub_timeout {
+    my ($self) = @_;
+    $self->{txt_grub_timeout}->set("-8");
```

Comment:
why `-8` shouldn't be `-1`, maybe it goes to the value we want because it is the minimum of the range, but better be specific as the user cannot type that afir, only the libyui-rest-api can.

Reply:
OMG that typo stayed right under my eyes the whole time, thanks for spotting it !